web-comp.js

```diff
@@ -0,0 +1,204 @@
+const catalogProducts = [
+    {
+        id: 1,
+        imageURL: "images/product/ak101-img.jpeg",
+        altAlert: " it's an ak101",
```

Comment:
Probably that's my fault, change this sinc altAlert doesn't make sense. It has been generated by chatgpt when I created the array.

Reply:
should i remove the alt from the object generation, or just change the alt?

Reply:
fixed